test/unit/rules/no-this-in-template-only-components-test.js

```diff
@@ -61,6 +71,8 @@ generateRuleTests({
     {
       template: '{{input id=(concat this.elementId "-username")}}',
       result: {
+        filePath: 'app/components/foo.hbs',
```

Comment:
filePath may be absolute fs path

Reply:
Ya, but I think we can't really fix this until workingDir lands.